Sources/MeiliSearch/Tasks.swift

```diff
@@ -49,21 +41,34 @@ struct Tasks {
 
   // get all on client
   func getAll(
+    params: TasksQuery? = nil,
     _ completion: @escaping (Result<Results<Task>, Swift.Error>) -> Void) {
-      getAll(path: "/tasks", completion)
+      getAll(path: "/tasks", params: params, completion)
   }
 
   // get all on index
   func getAll(
     uid: String,
+    params: TasksQuery? = nil,
     _ completion: @escaping (Result<Results<Task>, Swift.Error>) -> Void) {
-      getAll(path: "/indexes/\(uid)/tasks", completion)
+      var query: TasksQuery?
+
+      if params != nil {
+        params?.indexUid.append(uid)
```

Comment:
In case we enter this function through the `client.getTasks` and not via the `index.getTasks` the indexUid would be overwritten

Reply:
I've answered here: https://github.com/meilisearch/meilisearch-swift/pull/315#discussion_r936755487 from my POV this cannot happen! :)